scripts/qa_crowbarsetup.sh

```diff
@@ -2848,14 +2848,14 @@ function custom_configuration()
                 proposal_set_value nova default "['deployment']['nova']['elements']['${role_prefix}-controller']" "['cluster:$clusternameservices']"
 
                 # only use remaining nodes as compute nodes, keep cluster nodes dedicated to cluster only
-                local novanodes=$unclustered_nodes
+                local novanodes=($unclustered_nodes)
 
                 # make sure we do not pick SP1 nodes on cloud7
                 if [ -n "$deployceph" ] && iscloudver 7 ; then
                     novanodes=$unclustered_sles12plusnodes
                 fi
 
-                if [ -z $novanodes ]; then
+                if [ -z "$novanodes" ]; then
```

Comment:
Done. I guess we wouldn't strictly need the quotes now anymore but I left them in place.